### PR TITLE
Attempt to handle large product queries more gracefully.

### PR DESCRIPTION
**Pull request by bryanjnelson**
- Create/handle new Session Expired Error
- Lower max batch size of product lower to 25 from 100

### CODE REVIEW

File: src/components/product/product.loader.ts [hunk 1]
```diff
@@ -23,4 +27,12 @@ export class ProductLoader extends OrderedNestDataLoader<AnyProduct> {
   async loadMany(ids: readonly ID[]) {
     return await this.products.readMany(ids, this.session);
   }
+
+  getOptions(): OrderedNestDataLoaderOptions<AnyProduct> {
+    return {
+      // Increase the batching timeframe from the same nodejs frame to 10ms
+      batchScheduleFn: (cb) => setTimeout(cb, 10),
```
Comment (CarsonF) [inline src/components/product/product.loader.ts:34]:
```suggestion
      ...super.getOptions(),
```

File: src/core/database/errors.ts [hunk 2]
```diff
@@ -55,6 +55,27 @@ export class ServiceUnavailableError extends Neo4jError {
   }
 }
 
+export class SessionExpiredError extends Neo4jError {
+  static readonly code = 'ServiceUnavailable' as const;
```
Comment (CarsonF) [inline src/core/database/errors.ts:59]:
```suggestion
  static readonly code = 'SessionExpired' as const;
```
Also need to call `fromNeo` in the `cast` function below.

File: src/core/exception.filter.ts [hunk 3]
```diff
@@ -142,6 +146,12 @@ export class ExceptionFilter implements GqlExceptionFilter {
         message: 'Failed to retrieve data from CORD database',
       };
     }
+    if (exs.some((e) => e instanceof SessionExpiredError)) {
+      return {
+        codes: ['SessionExpired', 'Transient', 'Database', 'Server'],
+        message: 'The CORD database session has expired',
```
Comment (CarsonF) [inline src/core/exception.filter.ts:152]:
```suggestion
        message: 'The query to the database has expired',
```
maybe this would be better?

**Review by CarsonF — CHANGES_REQUESTED** [review]

**Comment by bryanjnelson** [comment]
@CarsonF - I'll squash after your review.

File: src/core/database/errors.ts [hunk 4]
```diff
@@ -55,6 +55,27 @@ export class ServiceUnavailableError extends Neo4jError {
   }
 }
 
+export class SessionExpiredError extends Neo4jError {
+  static readonly code = 'SessionExpiredError' as const;
```
Comment (CarsonF) [inline src/core/database/errors.ts:59]:
```suggestion
  static readonly code = 'SessionExpired' as const;
```

**Review by CarsonF — COMMENTED** [review]
Close ;)

File: src/core/database/errors.ts (hunk 4 above)
Comment (bryanjnelson) [reply src/core/database/errors.ts:59]: 🤦🏼‍♂️

**Review by CarsonF — APPROVED** [review]
Thanks man!

**Comment by bryanjnelson** [comment]
@CarsonF - it's forcing another review 😕

**Review by CarsonF — APPROVED** [review]

**Pull request merged**